### PR TITLE
Move old user conversion down slightly (fixes #8050)

### LINKED ISSUE #8050: Cannot read field "proxies"
### Stack trace

> This is my `latest.log` file
```
[22:53:34] [ServerMain/INFO]: Building unoptimized datafixer
[22:53:36] [ServerMain/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[22:53:36] [ServerMain/INFO]: Found new data pack file/bukkit, loading it automatically
[22:53:38] [ServerMain/INFO]: Loaded 7 recipes
[22:53:40] [Server thread/INFO]: Starting minecraft server version 1.19
[22:53:40] [Server thread/INFO]: Loading properties
[22:53:40] [Server thread/INFO]: This server is running Paper version git-Paper-36 (MC: 1.19) (Implementing API version 1.19-R0.1-SNAPSHOT) (Git: 5593f4e)
[22:53:41] [Server thread/INFO]: Server Ping Player Sample Count: 12
[22:53:41] [Server thread/INFO]: Using 4 threads for Netty based IO
[22:53:41] [Server thread/ERROR]: Encountered an unexpected exception
java.lang.NullPointerException: Cannot read field "proxies" because the return value of "io.papermc.paper.configuration.GlobalConfiguration.get()" is null
	at net.minecraft.server.players.OldUsersConverter.lookupPlayers(OldUsersConverter.java:70) ~[?:?]
	at net.minecraft.server.players.OldUsersConverter.convertOpsList(OldUsersConverter.java:214) ~[?:?]
	at net.minecraft.server.dedicated.DedicatedServer.convertOldUsers(DedicatedServer.java:653) ~[paper-1.19.jar:git-Paper-36]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:213) ~[paper-1.19.jar:git-Paper-36]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1121) ~[paper-1.19.jar:git-Paper-36]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:302) ~[paper-1.19.jar:git-Paper-36]
	at java.lang.Thread.run(Thread.java:831) ~[?:?]
[22:53:41] [Server thread/WARN]: Failed to get system info for World Generation
java.lang.NullPointerException: Cannot invoke "net.minecraft.world.level.storage.SaveData.B()" because "this.m" is null
	at net.minecraft.server.MinecraftServer.lambda$fillSystemReport$15(MinecraftServer.java:1692) ~[paper-1.19.jar:git-Paper-36]
	at net.minecraft.SystemReport.setDetail(SystemReport.java:61) ~[paper-1.19.jar:git-Paper-36]
	at net.minecraft.server.MinecraftServer.fillSystemReport(MinecraftServer.java:1691) ~[paper-1.19.jar:git-Paper-36]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1213) ~[paper-1.19.jar:git-Paper-36]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:302) ~[paper-1.19.jar:git-Paper-36]
	at java.lang.Thread.run(Thread.java:831) ~[?:?]
[22:53:41] [Server thread/ERROR]: This crash report has been saved to: /home/minecraft/./crash-reports/crash-2022-06-25_22.53.41-server.txt
[22:53:41] [Server thread/INFO]: Stopping server
[22:53:41] [Server thread/INFO]: Saving players
[22:53:41] [Server thread/INFO]: Saving worlds
[22:53:41] [Server thread/INFO]: ThreadedAnvilChunkStorage: All dimensions are saved
[22:53:41] [Server thread/INFO]: Flushing Chunk IO
[22:53:41] [Server thread/INFO]: Closing Thread Pool
[22:53:41] [Server thread/INFO]: Closing Server
```
> My crash log file
```
---- Minecraft Crash Report ----
// You should try our sister game, Minceraft!

Time: 6/25/22, 10:53 PM
Description: Exception in server tick loop

java.lang.NullPointerException: Cannot read field "proxies" because the return value of "io.papermc.paper.configuration.GlobalConfiguration.get()" is null
	at net.minecraft.server.players.OldUsersConverter.lookupPlayers(OldUsersConverter.java:70)
	at net.minecraft.server.players.OldUsersConverter.convertOpsList(OldUsersConverter.java:214)
	at net.minecraft.server.dedicated.DedicatedServer.convertOldUsers(DedicatedServer.java:653)
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:213)
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1121)
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:302)
	at java.base/java.lang.Thread.run(Thread.java:831)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.19
	Minecraft Version ID: 1.19
	Operating System: Linux (amd64) version 5.4.0-121-generic
	Java Version: 17-ea, Oracle Corporation
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Oracle Corporation
	Memory: 854528544 bytes (814 MiB) / 1073741824 bytes (1024 MiB) up to 1073741824 bytes (1024 MiB)
	CPUs: 16
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Xeon(R) CPU E5-2650 0 @ 2.00GHz
	Identifier: Intel64 Family 6 Model 45 Stepping 7
	Microarchitecture: Sandy Bridge (Server)
	Frequency (GHz): 2.00
	Number of physical packages: 1
	Number of physical CPUs: 8
	Number of logical CPUs: 16
	Graphics card #0 name: unknown
	Graphics card #0 vendor: unknown
	Graphics card #0 VRAM (MB): 0.00
	Graphics card #0 deviceId: unknown
	Graphics card #0 versionInfo: unknown
	Virtual memory max (MB): 18080.42
	Virtual memory used (MB): 3650.58
	Swap memory total (MB): 2048.00
	Swap memory used (MB): 0.00
	JVM Flags: 3 total; -XX:+UseG1GC -Xms1G -Xmx1G
	CraftBukkit Information: 
   Running: Paper version git-Paper-36 (MC: 1.19) (Implementing API version 1.19-R0.1-SNAPSHOT) false
   Plugins: {}
   Warnings: DEFAULT
   Reload Count: 0
   Threads: { TIMED_WAITING Craft Scheduler Thread - 3: [java.base@17-ea/jdk.internal.misc.Unsafe.park(Native Method), java.base@17-ea/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252), java.base@17-ea/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401), java.base@17-ea/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1060), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1121), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635), java.base@17-ea/java.lang.Thread.run(Thread.java:831)], WAITING Worker-Main-4: [java.base@17-ea/jdk.internal.misc.Unsafe.park(Native Method), java.base@17-ea/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341), java.base@17-ea/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506), java.base@17-ea/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3454), java.base@17-ea/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3425), java.base@17-ea/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623), java.base@17-ea/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1121), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635), java.base@17-ea/java.lang.Thread.run(Thread.java:831)], WAITING Worker-Main-6: [java.base@17-ea/jdk.internal.misc.Unsafe.park(Native Method), java.base@17-ea/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341), java.base@17-ea/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506), java.base@17-ea/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3454), java.base@17-ea/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3425), java.base@17-ea/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623), java.base@17-ea/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1121), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635), java.base@17-ea/java.lang.Thread.run(Thread.java:831)], RUNNABLE Notification Thread: [], TIMED_WAITING Craft Scheduler Thread - 1: [java.base@17-ea/jdk.internal.misc.Unsafe.park(Native Method), java.base@17-ea/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252), java.base@17-ea/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401), java.base@17-ea/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1060), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1121), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635), java.base@17-ea/java.lang.Thread.run(Thread.java:831)], TIMED_WAITING process reaper: [java.base@17-ea/jdk.internal.misc.Unsafe.park(Native Method), java.base@17-ea/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252), java.base@17-ea/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401), java.base@17-ea/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1060), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1121), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635), java.base@17-ea/java.lang.Thread.run(Thread.java:831)], WAITING Log4j2-AsyncAppenderEventDispatcher-1-Async: [java.base@17-ea/jdk.internal.misc.Unsafe.park(Native Method), java.base@17-ea/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341), java.base@17-ea/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506), java.base@17-ea/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3454), java.base@17-ea/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3425), java.base@17-ea/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623), java.base@17-ea/java.util.concurrent.ArrayBlockingQueue.take(ArrayBlockingQueue.java:420), org.apache.logging.log4j.core.appender.AsyncAppenderEventDispatcher.dispatchAll(AsyncAppenderEventDispatcher.java:71), org.apache.logging.log4j.core.appender.AsyncAppenderEventDispatcher.run(AsyncAppenderEventDispatcher.java:63)], RUNNABLE DestroyJavaVM: [], RUNNABLE Server thread: [java.base@17-ea/java.lang.Thread.dumpThreads(Native Method), java.base@17-ea/java.lang.Thread.getAllStackTraces(Thread.java:1647), org.bukkit.craftbukkit.v1_19_R1.CraftCrashReport.get(CraftCrashReport.java:33), org.bukkit.craftbukkit.v1_19_R1.CraftCrashReport.get(CraftCrashReport.java:17), net.minecraft.SystemReport.a(SystemReport.java:61), net.minecraft.CrashReport.<init>(CrashReport.java:36), net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1270), net.minecraft.server.MinecraftServer.v(MinecraftServer.java:1211), net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:302), net.minecraft.server.MinecraftServer$$Lambda$4374/0x00000008014e5978.run(Unknown Source), java.base@17-ea/java.lang.Thread.run(Thread.java:831)], WAITING Worker-Main-5: [java.base@17-ea/jdk.internal.misc.Unsafe.park(Native Method), java.base@17-ea/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341), java.base@17-ea/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506), java.base@17-ea/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3454), java.base@17-ea/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3425), java.base@17-ea/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623), java.base@17-ea/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1121), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635), java.base@17-ea/java.lang.Thread.run(Thread.java:831)], TIMED_WAITING Craft Scheduler Thread - 2: [java.base@17-ea/jdk.internal.misc.Unsafe.park(Native Method), java.base@17-ea/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252), java.base@17-ea/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401), java.base@17-ea/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1060), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1121), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635), java.base@17-ea/java.lang.Thread.run(Thread.java:831)], RUNNABLE Reference Handler: [java.base@17-ea/java.lang.ref.Reference.waitForReferencePendingList(Native Method), java.base@17-ea/java.lang.ref.Reference.processPendingReferences(Reference.java:253), java.base@17-ea/java.lang.ref.Reference$ReferenceHandler.run(Reference.java:215)], WAITING Finalizer: [java.base@17-ea/java.lang.Object.wait(Native Method), java.base@17-ea/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155), java.base@17-ea/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:176), java.base@17-ea/java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:171)], WAITING Worker-Main-2: [java.base@17-ea/jdk.internal.misc.Unsafe.park(Native Method), java.base@17-ea/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341), java.base@17-ea/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506), java.base@17-ea/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3454), java.base@17-ea/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3425), java.base@17-ea/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623), java.base@17-ea/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1121), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635), java.base@17-ea/java.lang.Thread.run(Thread.java:831)], WAITING Worker-Main-3: [java.base@17-ea/jdk.internal.misc.Unsafe.park(Native Method), java.base@17-ea/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341), java.base@17-ea/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506), java.base@17-ea/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3454), java.base@17-ea/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3425), java.base@17-ea/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623), java.base@17-ea/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1121), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635), java.base@17-ea/java.lang.Thread.run(Thread.java:831)], TIMED_WAITING Craft Scheduler Thread - 0: [java.base@17-ea/jdk.internal.misc.Unsafe.park(Native Method), java.base@17-ea/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252), java.base@17-ea/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401), java.base@17-ea/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1060), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1121), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635), java.base@17-ea/java.lang.Thread.run(Thread.java:831)], TIMED_WAITING Common-Cleaner: [java.base@17-ea/java.lang.Object.wait(Native Method), java.base@17-ea/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155), java.base@17-ea/jdk.internal.ref.CleanerImpl.run(CleanerImpl.java:140), java.base@17-ea/java.lang.Thread.run(Thread.java:831), java.base@17-ea/jdk.internal.misc.InnocuousThread.run(InnocuousThread.java:161)], RUNNABLE Signal Dispatcher: [], TIMED_WAITING Timer hack thread: [java.base@17-ea/java.lang.Thread.sleep(Native Method), net.minecraft.SystemUtils$7.run(SystemUtils.java:706)], WAITING Worker-Main-1: [java.base@17-ea/jdk.internal.misc.Unsafe.park(Native Method), java.base@17-ea/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341), java.base@17-ea/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506), java.base@17-ea/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3454), java.base@17-ea/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3425), java.base@17-ea/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623), java.base@17-ea/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1121), java.base@17-ea/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635), java.base@17-ea/java.lang.Thread.run(Thread.java:831)], WAITING Java2D Disposer: [java.base@17-ea/java.lang.Object.wait(Native Method), java.base@17-ea/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155), java.base@17-ea/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:176), java.desktop@17-ea/sun.java2d.Disposer.run(Disposer.java:144), java.base@17-ea/java.lang.Thread.run(Thread.java:831)],}
   
   Force Loaded Chunks: {}
	Server Running: true
	Player Count: 0 / 16; []
	Data Packs: vanilla, file/bukkit (incompatible)
	World Generation: ERR
	Is Modded: Definitely; Server brand changed to 'Paper'
	Type: Dedicated Server (map_server.txt)
```

### Plugin and Datapack List

AngelChest 9.2.0
EditableSigns 1.2.0
ItemEdit 3.0.3
Item_Effects 1.1.3
oneplayersleep 3.3
OverleveledEnchanter 2.7
SilkSpawners v2
SkinFixer 1.7.6
Stormer's Homes 0.1.5
Waypoints 5.0-beta1

### Actions to reproduce (if known)

I've used papermc in a custom docker image for the last several years (well more like since Minecraft 1.11.2).

I've used the same custom docker image to play papermc versions:

* 1.11.2
* 1.12
* 1.12.2
* 1.16
* And now trying to use it in 1.19 (I needed to modify it to have a newer java but except for that little bump it appears to be working)

I'm not sure if my end I can provide a step by step guide,
I just know I don't even get to loading plugins anyway.

> Hence is why I'm trying to use OverloadedEnchanter (which if I recall it's specs only supports Minecraft 1.18.2 and a few versions lower)

At first I thought I found issues referring to my issue 
(most of it saying to blow away the configs as they were invalid, but unfortunately that now leads me to only having a empty config directory)

> Should have put all this probably in the other section but oops, oh well.

### Paper version

This server is running Paper version git-Paper-36 (MC: 1.19) (Implementing API version 1.19-R0.1-SNAPSHOT) (Git: 5593f4e)

### Other

_No response_

### PR DESCRIPTION
**Pull request by NoahvdAa**
Fixes #8050 by moving the old user convertion down slightly (or the config and deobf up, depending on how you look at it).

If you want to test this, turn your server to offline mode and then simply make a file `white-list.txt` and put a player name in it.

### CODE REVIEW

**Review by Machine-Maker — APPROVED** [review]

File: patches/server/0596-fix-converting-txt-to-json-file.patch [hunk 1]
```diff
@@ -21,22 +21,22 @@ index d83bed436d2ad51cef83ecbf0c7df227a67ff404..dc96b30c70cd79d7b2a0322f32b9399a
...
-@@ -199,6 +199,12 @@ public class DedicatedServer extends MinecraftServer implements ServerInterface
-         org.spigotmc.SpigotConfig.init((java.io.File) options.valueOf("spigot-settings"));
-         org.spigotmc.SpigotConfig.registerCommands();
-         // Spigot end
-+        // Paper start - moved up to right after PlayerList creation but before file load/save
+@@ -203,6 +203,12 @@ public class DedicatedServer extends MinecraftServer implements ServerInterface
+         io.papermc.paper.util.ObfHelper.INSTANCE.getClass(); // Paper - load mappings for stacktrace deobf and etc.
+         paperConfigurations.initializeGlobalConfiguration();
+         paperConfigurations.initializeWorldDefaultsConfiguration();
++        // Paper - moved up to right after PlayerList creation but before file load/save
```
Comment (kennytv) [inline patches/server/0596-fix-converting-txt-to-json-file.patch]: start

Comment (NoahvdAa) [reply patches/server/0596-fix-converting-txt-to-json-file.patch]: Thank you, fixed

**Comment by NoahvdAa** [comment]
Rebased

**Comment by NoahvdAa** [comment]
Rebased!

**Review by electronicboy — APPROVED** [review]

**Pull request merged**